src/test/java/org/folio/marc4ld/mapper/test/Marc2Bibframe650IT.java

```diff
@@ -0,0 +1,98 @@
+package org.folio.marc4ld.mapper.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.List;
+import java.util.stream.StreamSupport;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.dictionary.model.ResourceEdge;
+import org.folio.marc4ld.service.marc2ld.Marc2BibframeMapperImpl;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@EnableConfigurationProperties
+@SpringBootTest(classes = SpringTestConfig.class)
+class Marc2Bibframe650IT {
+
+  @Autowired
+  private Marc2BibframeMapperImpl marc2BibframeMapper;
+
+  @Test
+  void map_shouldContains_severalRegion() {
+    // given
+    var marc = loadResourceAsString("fields/marc_650.jsonl");
+
+    // when
+    var resource = marc2BibframeMapper.fromMarcJson(marc);
+
+    System.out.println(resource);
+
+    assertThat(resource)
+      .isNotNull();
+
+    var subject = geSubject(resource);
+    assertThat(subject)
```

Comment:
Can we also verify that two outgoing "subFocus" edges are created? One for "Italy" and the second for "Florence"

Reply:
a specially create 2 separate test
This is for control properties
next (in this file) for edges `map_shouldReturn_severalGeographicalNodes`